xdsl/printer.py

```diff
@@ -652,7 +652,7 @@ def _print_attr_string(self, attr_tuple: tuple[str, Attribute]) -> None:
             self.print(f'"{attr_tuple[0]}" = ')
             self.print_attribute(attr_tuple[1])
 
-    def _print_op_attributes(self, attributes: Dict[str, Attribute]) -> None:
+    def print_op_attributes(self, attributes: Dict[str, Attribute]) -> None:
```

Comment:
@math-fehr I made this API public so custom ops can print the default attribute dict as well.

Will push another PR to adopt this in the rest of the codebase.

Reply:
See #1171

Reply:
Yes!